thrift/matching.thrift

```diff
@@ -109,6 +109,10 @@ struct DescribeTaskListRequest {
   20: optional shared.DescribeTaskListRequest descRequest
 }
 
+struct GetTaskListsByDomainRequest {
+  10: optional string domain
```

Comment:
Is this domainUUID or domainName? if is domainName, can we reuse the one defined in shared? If's UUID, shall we make it more clear in the naming?

Reply:
It is domain name here

Reply:
updated

Reply:
updated